decode_morse.rb

```diff
@@ -0,0 +1,16 @@
+def decode_character(letter)
+  dictionary = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+                 '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+                 '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+                 '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z' }
+  dictionary[letter]
+end
+
+def decode_word(word)
+  word.split(/ /).map { |letter| decode_character letter }.join
+end
+
+def decode_sentence(sentence)
+  sentence.split(/ /).map { |word| decode_word word }.join(' ')
```

Comment:
- [ ] Kindly consider adding one more space in the split method to split between words correctly because the output of the following `p decode_sentence(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")`, it will be `"A   B O X   F U L L    O F    R U B I E S"`.
Your code will be like this:
```ruby
def decode_sentence(sentence)
  sentence.split(/  /).map { |word| decode_word word }.join(' ')
end
```

Reply:
Hello @omarsalem7 , thank you for the suggestion. Implemented.